src/StaticArrays.jl

```diff
@@ -50,7 +50,7 @@ include("inv.jl")
 include("eigen.jl")
 include("cholesky.jl")
 
-include("FixedSizeArrays.jl")
+# include("FixedSizeArrays.jl")
```

Comment:
Sorry, @andreasnoack and i got a bit sidetracked... My only complaint was the `where`, since its not 0.5 compatible... I was fine with just removing the whole thing for now. Porting geometrytypes ( which uses this) to 0.6 will be a bit of work anyways... Lets just seperate it!